exercises/15_prime.rb

```diff
@@ -0,0 +1,15 @@
+def generate_prime(limit)
+  result = 1.step(limit, 2).filter do |num|
```

Comment:
Use .select instead of .filter as filter is only available in newer versions of ruby( > 2.5). Both do the same work but .select works on all ruby versions